docs/Actions/inject-data.md

```diff
@@ -0,0 +1,273 @@
+---
+id: inject-data
+title: Inject Data
+sidebar_label: Inject Data
+---
+
+> Audience: [`Business Users`](/docs/audience#business-users)<br/>
+> Skill Prerequisites: `None`
+
+An action that lets you add data as tokens to the current execution context. Inject data can be used to load data that other actions other actions down the stack need.
+
+## `Typical Use Cases`
+
+- Generate a new token
+- Concatenate 2 tokens
+- Overwrite a token
+
+## `Don't use it to`
+
+- Execute a Variables token - use [Apply Tokens](/docs/Actions/apply-tokens.md) or [Execute Tokens](/docs/Actions/execute-tokens.md) actions instead
+- Apply tokens inside a text from different sources - use [Apply Tokens](/docs/Actions/apply-tokens.md) or [Execute Tokens](/docs/Actions/execute-tokens.md) actions instead
+- Use a token into another action list, only use it for the current one - store the data in a [hidden field](/docs/Actions/hidden-field.md), cookies or server session instead
+
+## `Related Actions`
+
+| Action Name | Description|
+|-------------|------------|
+| [Run SQL Query](/docs/Actions/run-sql-query.md) | Executes an SQL statement, optionally capturing the output |
+
+## `Input Parameter Reference`
+
+| Parameter     | Description                           | Supports Variables |
+|---------------|---------------------------------------|--------------------|
+| Name/Value | First parameter refers to the name of the variable, the second one refers to the value assigned. The action use a key-value(name-value) concept. <br></br> Each data that you want to inject has different parameters. | Yes |
+
+## `Output Parameters Reference`
+
+| Parameter | Description |
+|-----------|-------------|
+| Output Token Name | The Name parameter is stored as a token for further use anywhere in the form. |
+
+## `Examples`
+
+### `1. Inject data using Run SQL Query`
+
+​
+The action below selects first client name from a database using Run SQL Query and sends it into the form. <br></br>[Import it](/docs/Actions/import-actions.md) into your application to see it in action.
```

Comment:
Let's prefer doing the examples over existing tables, such as Users. This will make the examples runnable regardless of setup.

Reply:
Fixed it